src/storages/types.ts

```diff
@@ -292,6 +292,11 @@ export interface IEventsCacheBase {
   track(data: SplitIO.EventData, size?: number): MaybeThenable<boolean>
 }
 
+export interface IImpressionCountsCacheBase {
```

Comment:
I think we can remove this interface, and use `IImpressionCountsCacheSync` for now, because ImpressionCountInRedis extends the one in memory